### DISCUSSION TITLE
How to use the firefox profile for another application?

### DISCUSSION THREAD

**Discussion by Raj2032** [netblue30/firejail#6508] [Q&A] [answered]
I want to use firefox's profile but for [bottles](https://github.com/bottlesdevs/Bottles) however not too sure if there is an easy command or something I can use to use firefox's profile for bottles?

**Comment by rusty-snake** [comment, marked as answer]
To run bottles with the firefox profile: `firejail --profile=firefox /usr/bin/bottles`

However I doubt that it works as bottles has different requirements that firefox.

**Reply by Raj2032** [reply]
Right I see, I suppose I should use bottles profile but add custom restrictions.
